analysis/README.md

```diff
@@ -0,0 +1,41 @@
+# Edna analysis
+
+This folder contains implementation of Edna data analysis in Python.
+
+## Usage
+
+First, normally you don't want to call this code directly, it is supposed to be called from `edna-server`.
+We use [`poetry`](https://python-poetry.org/) for dependency management, though you can install dependencies using your favorite way and not use `poetry`.
+To get shell with all dependencies installed run `poetry install` and then `poetry shell` from this directory.
+
+## Features
+
+The list of features implemented in an analytic module:
+1. Compute parameters of fitted sigmoid.
+2. Detect potential outliers.
+3. Compute parameters of fitted sigmoid without outliers.
+
+### Input format
+JSON containing list of experiments n the form:
+```
+["experiment": int,
+"data": [[float, float], ...]]
```

Comment:
Did you intentionally omit `{}` here and in output format?

Reply:
Dunno why I did it like this, will add.

Reply:
Shouldn't it be
```
[{"experiment": int,
"data": [[float, float], ...]}]
```
?
maybe even
```
[{"experiment": int,
"data": [[float, float], ...]}, ...]
```

Reply:
Oof, I'm kinda retarded, sorry. Sec